api/get_batches.go

```diff
@@ -19,26 +20,26 @@ func (a *API) GetBatches(from, to *models.Uint256) ([]dto.Batch, error) {
 	return batches, nil
 }
 
-func (a *API) unsafeGetBatches(from, to *models.Uint256) ([]dto.Batch, error) {
+func (a *API) unsafeGetBatches(from, to *models.Uint256) ([]interface{}, error) {
 	batches, err := a.storage.GetBatchesInRange(from, to)
 	if err != nil {
-		return []dto.Batch{}, err
+		return []interface{}{}, err
 	}
 
-	batchesWithSubmission := make([]dto.Batch, 0, len(batches))
+	batchesWithSubmission := make([]interface{}, 0, len(batches))
```

Comment:
We can keep  `batchesWithSubmission` type as `[]dto.Batch` as we append only objects of that type.

Reply:
Nice catch. It was a leftover from when I wanted to introduce `dto.SubmittedBatch` type.